### ISSUE TITLE
iOS version. 

### ISSUE THREAD

**Issue by kostetskydmitry** [whitedogg13/android-beam-rn-demo#1]
Hi, 

Thanks for sharing your good project. Would you please make an iOS version? I can't wait to see it. 

Thanks.

**Comment by whitedogg13** [comment]
Hi, Android Beam (NFC p2p mode) API is only available in Android, so it's not possible to do so.

Thanks

**Comment by kostetskydmitry** [comment]
So is it impossible to send String data between 2 iOS devices over NFC?

**Comment by kostetskydmitry** [comment]
I'd like to implement same feature in React Native iOS.

**Comment by whitedogg13** [comment]
It's not possible to do this in RN for iOS. You can consider using QRCode or BLE for this case.

**Comment by kostetskydmitry** [comment]
I don't have any knowledge of BLE. If you share sample code of BLE, then it will be really helpful. 

Thanks in advance.

**Comment by whitedogg13** [comment]
What's in the video isn't Android Beam, which is in NFC peer-to-peer mode. Alternatively it uses Android HCE (in NFC card emulation mode) API to make the Android phone to act as a NFC tag.

It's totally possible to do so, but currently our RN NFC library doesn't support HCE yet, as requested [here] (https://github.com/whitedogg13/react-native-nfc-manager/issues/79).

**Comment by kostetskydmitry** [comment]
Okay.

**Comment by kostetskydmitry** [comment]
Would you please share the RN BLE example which can communicate between 2 devices for both iOS and Android?

**Comment by whitedogg13** [comment]
Hi, I don't have a working example now, but using BLE should be possible.

However, I think your requirement can be easier implemented via a QR code builder / scanner. One phone encode the message into a QR code image, and another one scan it to decode the message.

Following package can help you quickly build such a feature:

* [react-native-qrcode](https://github.com/cssivision/react-native-qrcode): encode message into a dynamic QR code image
* [react-native-camera](https://github.com/react-native-community/react-native-camera): QR code scanner to decode the message

Hope that helps

**Comment by kostetskydmitry** [comment]
Yes, We have already solved that issue with QR code. But need to find another solution... 
If we use BLE, maybe users need to send connection request and accept it? After accepted, 2 device can communicate?

**Comment by whitedogg13** [comment]
I don't have a BLE-ready sample for now, sorry!
